sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1463,7 +1464,12 @@ def init(self):
         # Initialize xcvr table helper
         xcvr_table_helper = XcvrTableHelper()
 
-        self.load_media_settings()
+        output = subprocess.check_output("cat /proc/cmdline", shell=True, universal_newlines=True)
```

Comment:
Can you check the db entry instead?

cmdline will never change, so if a device booted up with fast-reboot, cmdline will always have that mark. If we do config reload or load minigraph, we need to take cold reboot procedure.

Reply:
Please take reference from syncd.sh

Reply:
@yxieca - made the requested changes, please review